packages/prices/utils/price.ts

```diff
@@ -137,6 +137,22 @@ const splitDecimal = (
 	};
 };
 
+const applyDecimal = (
+	afterDecimal: string,
+	decimalSeparator: string,
+	minorUnit: number
+): string => {
+	if ( afterDecimal ) {
```

Comment:
Do we need to pad decimals here regardless?

Create a test case or 2dp currency, with a given value of `10.2`. It should result in `10.20`

Reply:
After adding the additional test case, I realized that padding the decimals was needed. I add this accordingly.